### PR TITLE
move rabbitmq signing key

### PR DESCRIPTION
**Pull request by EmilienM**
Signing key moved to https://www.rabbitmq.com/rabbitmq-release-signing-key.asc

This patch aims to update the module.

### CODE REVIEW

**Comment by EmilienM** [comment]
I took over  https://github.com/puppetlabs/puppetlabs-rabbitmq/pull/475 so we can have a quick fix.

File: manifests/repo/apt.pp [hunk 1]
```diff
@@ -7,7 +7,7 @@
   $repos        = 'main',
   $include_src  = false,
   $key          = 'F78372A06FF50C80464FC1B4F7B8CEA6056E8E56',
```
Comment (alexjfisher) [inline manifests/repo/apt.pp]: @EmilienM Could you confirm whether you think `$key` needs updating too?

Comment (EmilienM) [inline manifests/repo/apt.pp]: updated.

**Comment by EmilienM** [comment]
too many PR try to fix it, I'll close mine.

**Comment by michaelklishin** [comment]
http://www.rabbitmq.com/rabbitmq-signing-key-public.asc is available again.

**Pull request closed without merging**